web-api/src/ingestion/routes.rs

```diff
@@ -68,6 +68,18 @@ pub(super) fn configure_service(config: &mut ServiceConfig) {
                 .route(web::get().to(get_document_candidates))
                 .route(web::put().to(set_document_candidates)),
         )
+        .service(
+            // this resource is deprecated and undocumented and will be removed in the future
+            web::resource("/documents/candidates")
```

Comment:
this had to move up as it was actually unreachable below the `/documents/{id}` routes. i also added tests for the regression.